Automattic/wp-calypso
# Help Center: Add articles feedback form

Related to #86531

## Proposed Changes

* This PR adds a small feedback form at the end of articles. More context #86531

![image](https://github.com/Automattic/wp-calypso/assets/52076348/d89646cc-9326-48a1-9957-24add377df75)


## Testing Instructions



* Use live link
* Navigate to any article in Help Center and scroll all the way down
* You should see the feedback 👍 or 👎 
* After selecting “yes” or “no”, the users should be able to optionally submit any free text
* With Track Vigilante (PCYsg-T9m-p2), check if the event `calypso_inlinehelp_article_feedback_click` is being logged
* Also Make sure to check atomic sites
* Results can be found in Crowd Signal (https://app.crowdsignal.com/), search for `HelpCenter articles feedback`
* The after submitting animation is from Crowd Signal, we can provide a custom message, will investigate what is the best.


## Pre-merge Checklist
